### PR TITLE
Fix handlers in node.py to follow naming convention

### PR DESCRIPTION
**Pull request by palango**
Part of #4735

## Description

This just updates `node.py`, the same need to be done for other handlers.

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/transfer/node.py [hunk 1]
```diff
@@ -510,7 +510,7 @@ def handle_block(chain_state: ChainState, state_change: Block) -> TransitionResu
     return TransitionResult(chain_state, events)
 
 
-def handle_chain_init(
+def handle_action_chain_init(
```
Comment (LefterisJP) [inline raiden/transfer/node.py]: nitpick question: shouldn't it be `handle_action_init_chain` ?

File: raiden/transfer/node.py [hunk 2]
```diff
@@ -592,7 +592,7 @@ def handle_new_token_network(
     return TransitionResult(chain_state, events)
 
 
-def handle_node_change_network_state(
+def handle_action_node_change_network_state(
```
Comment (LefterisJP) [inline raiden/transfer/node.py]: same here: What's the exact naming convention? The state change is `ActionChangeNodeNetworkState` so I would expect `handle_action_change_node_network_state`

File: raiden/transfer/node.py (hunk 1 above)
Comment (palango) [reply raiden/transfer/node.py]: Good point, I missed that.

File: raiden/transfer/node.py (hunk 2 above)
Comment (palango) [reply raiden/transfer/node.py]: Amended the changes.

**Pull request merged**